modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -461,6 +462,35 @@ class BashWrapperBuilder {
         }
     }
 
+    protected String getTaskMetadata() {
```

Comment:
Worth adding a unit tests also for this method

Reply:
it is tested here: https://github.com/nextflow-io/nextflow/blob/ade72e2194b58cdb023619dd8ca977268d3ad0eb/modules/nextflow/src/test/groovy/nextflow/executor/BashWrapperBuilderTest.groovy#L340